src/tilemaker/README.md

```diff
@@ -0,0 +1,49 @@
+# Tilemaker
+
+This container generates raster tiles using the results of an analysis job. It's run along with
+analysis jobs and uses the shapefiles exported by the analysis.
+
+The easiest way to run it locally is to load the `AnalysisJob` object you're interested in and
+call its `generate_tiles` method.  That won't run the tilemaker command but will print the right
+command to run to the logs, so you can copy it from there and run it.
+
+
+## Adding and Editing Styles
+
+The tiles are defined using Mapnik XML style files in the `styles` directory.
+Since the style XML files aren't templated, the path of the shapefile that's used as the
+data source is hard-coded to a file path in /data/.  The `run_tilemaker.sh` entrypoint script
+downloads the required shapefiles to the expected paths.
+
+For ease of editing and previewing, the styles are defined and maintained in a
+Mapbox Studio Classic project from which the XML style definitions are extracted by hand.
+
+Changes to the working files should be committed so that they're kept in sync with the actual
+tile styles being applied.
+
+### To open and edit the styles
+
+- Download the latest release of Mapbox Studio Classic from https://mapbox.s3.amazonaws.com/mapbox-studio/index.html (the [Mapbox download page](https://www.mapbox.com/mapbox-studio-classic/) links to an older, seemingly-broken version) and install it (for Linux,
+that just means unzipping it and running the `atom` command from the base directory).
+- Modify the hard-coded absolute paths in the following files to point to the actual locations
+on your machine.
+  ```
+  working_files/combined_data.tm2source/data.yml
+  working_files/combined_data.tm2source/data.xml
+  working_files/combined_styles.tm2/project.yml
+  working_files/combined_styles.tm2/project.xml
+  ```
+- In Mapbox Studio Classic, click "Styles & Sources" > "Browse" and select the
```

Comment:
I was presented with this splash page on initial login to mapbox studio:

![screen shot 2017-07-05 at 09 26 05](https://user-images.githubusercontent.com/1818302/27866365-0c517470-6164-11e7-89be-4acfa0c236c5.png)

Where I initially had to select "Blank Source" in the right-most column before getting to the editor view that I _think_ your directions are referencing:

![screen shot 2017-07-05 at 09 27 41](https://user-images.githubusercontent.com/1818302/27866409-398f3ba2-6164-11e7-98f8-9c527dd6fc7f.png)

The initial blue splash view cannot be returned to once you've started working with a project unless you logout and login to the application again. I think an additional screenshot or two showing these pages would be helpful here to orient the README user.